module1.js

```diff
@@ -121,10 +134,14 @@ function question1(name1) {
     "Hi, my name is {firstName} {lastName}. I am {age} years old and work as a {occupation} in {location}."
 */
 
-// function question7( personObj ) {
-//   // TODO
-//   return /* ??? */;
-// }
+
+function question7( personObj ) {
+  for (var i = 0; i<personObj.length; i++ ){
+  }
+   return "Hi, my name is" + " " + personObj.firstName + " " +  personObj.lastName + "." + " " + "I am" + " " + personObj.age + " " + "years old and work as a" + " " + personObj.occupation + " " + "in" + " " + personObj.location + "."
+
+  // TODO
+}
 
```

Comment:
Good work in accessing the object. If you had a much larger object, or a very long object name, you could assign it to a shorter function scoped variable, and access its properties that way. Something like this...
```
var obj = personObj;
return `Hi, my name is ${obj.firstName} etc...
```
this much leave you needing to do rather less typing. Also look at the Template Literals article mentioned on the previous question - it makes things much easier to read.

Reply:
thank you crise this is the fisrt time i filling happy because i see kindly person follow me step by step